packages/common/infra/src/orm/core/table.ts

```diff
@@ -194,20 +194,27 @@ export class Table<T extends TableSchemaBuilder> {
 
   find(where: FindEntityInput<T>): Entity<T>[] {
     return this.adapter.find({
-      where: Object.entries(where).map(([field, value]) => ({
-        field,
-        value,
-      })),
+      where: Object.entries(where)
+        .map(([field, value]) => ({
+          field,
+          value,
+        }))
+        .filter(({ value }) => value !== undefined),
     });
   }
 
   find$(where: FindEntityInput<T>): Observable<Entity<T>[]> {
     return new Observable<Entity<T>[]>(subscriber => {
       const unsubscribe = this.adapter.observe({
-        where: Object.entries(where).map(([field, value]) => ({
-          field,
-          value,
-        })),
+        where:
+          where === '*'
```

Comment:
this is no sense, you can make where parameter optional instead of a hardcoded special char